src/components/breadcrumb/breadcrumb.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '@mapbox/mr-ui/icon';
+import { createUniqueCrumbs } from './utils';
+
+export default class Breadcrumb extends React.Component {
+  render() {
+    const { currentPage, section, domain, site, subsite } = this.props;
+
+    const Link = (props) => (
+      <React.Fragment>
+        <a className="link" href={props.href}>
+          {props.children}
+        </a>
+        <span className="color-gray-light inline-block px6">
+          <Icon name="chevron-right" inline={true} />
+        </span>
+      </React.Fragment>
+    );
+
+    const createLinks = createUniqueCrumbs([
+      ...(domain !== false ? [domain] : []),
+      ...(site ? [site] : []),
+      ...(subsite ? [subsite] : []),
+      ...(section ? [section] : []),
+      ...(currentPage ? [currentPage] : [])
+    ]);
+
+    return createLinks.length > 1 ? (
+      <div
+        className="dr-ui--breadcrumb none block-mm px24 pt12"
```

Comment:
We're ok to set these padding values here because `<Breadcrumb>` is a layout component and is expected to be outside of the main `Content`, right?

Reply:
This padding was designed to fit nicely into PageLayout (as it appears now). If we think we'd use this component elsewhere, then we would probably want to remove the padding here and add a wrapper div (or something) to add the necessary padding when this component is referenced in PageLayout.

Reply:
🦆 I think we should remove the padding to keep this component independent. I'll update!

Reply:
🦆 Or maybe, a theme prop to be able to pass classnames to the container!

Reply:
It's beautiful!